CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # CHANGELOG
 
 
+## N.N.N
+* issue #1225: Look For Solutions To Reconnect CMS Web Socket When It Disconnects.
```

Comment:
@tbone1020 - please use an issue number if it is referring to a ticket created in this repo. e.g., https://github.com/triniti/cms-js/issues/232